src/bootstrap.js

```diff
@@ -0,0 +1,42 @@
+var ZoteroOCR;
+
+function log(msg) {
+	Zotero.debug("Zotero OCR: " + msg);
+}
+
+function install() {
+	log("Installed Zotero OCR");
+}
+
+async function startup({ id, version, rootURI }) {
+	log("Starting Zotero OCR");
+	
+	Zotero.PreferencePanes.register({
+		pluginID: 'zotero-ocr@bib.uni-mannheim.de',
+		src: rootURI + 'prefs.xhtml',
+		//scripts: [rootURI + 'prefs.js']
+	});
+	
```

Comment:
```suggestion

```

Reply:
This line (and some more in other files) add trailing whitespace which should be removed.

Reply:
OK